Podfile

```diff
@@ -11,9 +11,9 @@ plugin 'cocoapods-art', :sources => [
   'tomtom-sdk-cocoapods'
 ]
 
-TOMTOM_SDK_VERSION = '0.2.3404'
```

Comment:
is there a hard requirement between the features newly added and this version upgrade ? 
if not I would rather leave renovate to update this. it would provide a perfect test scenario.

Reply:
There is a requirement based on this version.